app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Api::V1::CommentsController < ApplicationController
+  before_action :authorize_request
+  protect_from_forgery with: :null_session
+
+  def index
+    @comments = Comment.where(post_id: params[:post_id])
+
+    if @comments.empty?
+      json_response({ msg: 'No post, comments or user found' }, 400)
+    else
+      render json: @comments
+    end
+  end
+
+  #  def index
+  #   @user = current_user
+  #   @comments = @user.comments
+
+  #   respond_to do |format|
+  #     format.json { render json: @comments }
+  #   end
+  # end
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.